docs/reference/api/connectivity/lorawan/lorawanconfig.md

```diff
@@ -0,0 +1,91 @@
+<h2 id="lorawan-configuration">Configuring Mbed LoRaWAN Stack</h2>
```

Comment:
I this content should go on this page: https://github.com/ARMmbed/Handbook/blob/new_engine/docs/reference/configuration/Connectivity.md

Do you mind if we move it there and make it match that format?

Reply:
I am afraid not. There are many reasons why not. For instance, LoRaWAN is non-IP that's why it is out of the Network Socket scope. Secondly, the purpose of this page is just to let user know what he can overwrite in terms of configuration and how to do it. I think the link you provided, is used for creating json config files for embedded example codes which are then exported to Online Compiler. We are not embedding any example in the docs at the moment (@AnttiKauppila will communicate the reason).

Reply:
@hasnainvirk Thanks for the response.

- The linked page is for all connectivity, not just IP.
- If you like, we can add an empty line between the components, the way we did between the general purpose file system and LittleFS-specific configs: https://github.com/ARMmbed/Handbook/blob/new_engine/docs/reference/configuration/Storage.md
- I don't understand your next point. The purpose of the page I linked is also to let users know what they can overwrite in terms of configuration. Could you please elaborate?
- Thank you and @AnttiKauppila for clarifying about the examples. It makes sense not to include private examples.